.github/workflows/ci.yml

```diff
@@ -19,7 +19,6 @@ jobs:
       fail-fast: false
       matrix:
         deno:
-          - v1.x
```

Comment:
If we drop v1 CI, we don't what packages work in v1 anymore (any package could randomly break in v1 in the future development)